### ISSUE TITLE
[A11y][TFM]Name defined for the buttons 'shows and hides TFM filters' is not descriptive enough

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9413]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1745270

Environment Details:
Application Name: Filtering by Target Frameworks
#URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Windows Version: Win11
OS Version: 22621.1105
Chromium Edge Version: Version 109.0.1518.78 (Official build) (64-bit)

Repro Steps: 
Hit the URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Tab to Frameworks section.
Tab to the collapse/expand buttons present in the Frameworks section.
Inspect the name of the buttons using developer tools options (ctrl+shift+I).
Observe that name for all the collapse/expand buttons is 'shows and hides TFM filters' which is not descriptive enough.

Actual Result:
Name defined for the buttons 'shows and hides TFM filters' is not descriptive enough as there are 4 buttons with the same name in frameworks section.

Expected Result:
Name defined for the buttons 'shows and hides TFM filters' should be descriptive enough as there are 4 buttons with the same name in frameworks section.

Notes:
I think these buttons should read "Shows and Hides TFM filters for {Framework}" assuming that I have understood the feature correctly. Updating labels should be sufficient to fix this.
